flake.nix

```
@@ -0,0 +1,34 @@
+{
+  description = "snazzy terminal (suckless + lightweight)";
+
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system};
+      in rec {
+        packages = flake-utils.lib.flattenTree {
+          st-snazzy = pkgs.callPackage ./default.nix { };
+        };
+        defaultPackage = packages.st-snazzy;
+        apps.st-snazzy = flake-utils.lib.mkApp {
+          drv = packages.st-snazzy;
+          exePath = "/bin/st";
+        };
+        defaultApp = apps.st-snazzy;
```

```suggestion
        apps.default = apps.st-snazzy;
```

@sioodmy For some reason `defaultApp` didn't work for me, I had to switch it to `apps.default`. Can you confirm if this works for you?

Besides that this PR is all good.